plugins/incremental-ingestion-backend/migrations/20220906161750_add-ingestion-indexes.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
```

Comment:
Oh I see, there's multiple migrations.

I realise that this probably comes from your internal version, and you have it already installed. But it would be really neat if we could clean these up for the open source world so that they have a good initial state. 

BEST case: strip this down to be just one single migration file with the actual desired final state. Problem for you then is, you'll have to do a migration thing of some sort. Maybe hacking the migrations table by hand or something when switching to the open source provider, could make it easier.

Less optimal case: At least strip out things that we know are useless, such as creating ingestion.current_entities and then immediately removing it again. Imagine adding support for another database engine in the future; rewriting all of these little things in here and making them work across architectures just adds burden that goes to waste just to appease the database engine for a millisecond. :)

Reply:
As I'm sure you can tell, I'm consolidating migrations into a single file. Hopefully you can review it again when it's all done.